tests/pipeline/test_extraction.py

```diff
@@ -231,8 +229,8 @@ def test_run_psfex( decam_example_reduced_image_ds ):
         tmppsfxmlfile.unlink()
 
         psf = detector._run_psfex( tempname, sourcelist.image_id, psf_size=26 )
-        assert psf._header['PSFAXIS1'] == 29
-        assert psf._header['PSFAXIS1'] == 29
+        assert psf._header['PSFAXIS1'] == 27
+        assert psf._header['PSFAXIS1'] == 27
```

Comment:
All of these will need to be reset when you remove the psf_sampling clamping to a minimum of 1 in detection.py.